src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -185,8 +194,8 @@ describe('Test suite for dictionary modal', () => {
       errors: {},
     });
 
-    await wrapper3.instance().onSubmit(preventDefault);
-    expect(wrapper3.instance().state.errors).toEqual({});
+    await wrapper.instance().onSubmit(preventDefault);
```

Comment:
Same as above

Reply:
I had refactored the structure of the tests to align with the coding convention